.circleci/config.yml

```diff
@@ -492,7 +492,7 @@ jobs:
       - run:
           name: Build development contracts
           command: |
-            docker run --volumes-from with_code cosmwasm/workspace-optimizer:0.12.11
+            docker run --volumes-from with_code cosmwasm/workspace-optimizer:0.12.9
```

Comment:
```suggestion
            docker run --volumes-from with_code cosmwasm/workspace-optimizer:0.12.11
```

Reply:
this is fine to bump, the issue was with the docker -arm  version in optimize.sh which breaks CI.